changelog/2023-09-10-addedInsertPythonAction.md

```diff
@@ -0,0 +1,7 @@
+---
+tags: [enhancement, talon]
+mergeDate: 2023-09-10
+pullRequest: https://github.com/cursorless-dev/cursorless/pull/1875
```

Comment:
What if we use the PR number as the filename instead of specifying it in the file?

Reply:
@auscompgeek the plan is to have a bot maintain these, so having it in the content is easier (otherwise you have to do a file rename). See https://github.com/cursorless-dev/cursorless/issues/1796

the thinking behind having a bot maintained these is to avoid a two step dance: fix create the PR so that you know what the PR number is, then add another commit afterwards for the changelog.

@AndreasArvidsson I would argue that just `pullRequest: #1875` suffices

Reply:
> What if we use the PR number as the filename instead of specifying it in the file?

Interesting idea. I'd still want it in the front matter as well tho I think for ease of machine readability

It's possible a feature might come in in multiple PRs. Not sure how to handle that case